src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -598,3 +600,27 @@ protected theorem lt_of_le_ne (x y : BitVec n) (h1 : x <= y) (h2 : ¬ x = y) : x
   let ⟨y, lt⟩ := y
   simp
   exact Nat.lt_of_le_of_ne
+
+@[simp] private theorem Nat.cast_two_pow {i : Nat} :(↑((2 : Nat)^i) : Int) = 2^i := by
+  induction i
+  case zero => rfl
+  case succ i ih =>
+    simp [Nat.pow_succ, Int.pow_succ, ih]
+
```

Comment:
This is a special case of `ofNat_pow` in `src/Init/Omega/Int.lean`.

Rather than another private lemma, could you replace both with an `Int.ofNat_pow` in `Init/Data/Int/Lemma.lean`?

Reply:
Done!